osu.Game/Overlays/ModSelection/ModSelection.cs

```diff
@@ -0,0 +1,520 @@
+                            TriangleScale = 8.0f,
+                        },
+                    }
+                },
+                new FlowContainer
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Direction = FlowDirection.VerticalOnly,
+                    Children = new Drawable[]
+                    {
+                        // upper container
+                        new Container
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                            Height = 0.2f,
+                            Children = new Drawable[]
+                            {
+                                new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Colour = new Color4(30, 30, 30, 128)
+                                },
+                                new FlowContainer
+                                {
+                                    // upper container
+                                    RelativeSizeAxes = Axes.Both,
+                                    Width = padded_width,
+                                    Direction = FlowDirection.VerticalOnly,
+                                    Anchor = Anchor.TopCentre,
+                                    Origin = Anchor.TopCentre,
+
+                                    Children = new Drawable[]
+                                    {
+                                        new SpriteText
+                                        {
+                                            Text = @"Gameplay Modes",
+                                            TextSize = 30,
+                                            Padding = new MarginPadding { Top = 10, Bottom = 8 },
+                                        },
+                                        new SpriteText
+                                        {
+                                            Text = @"Mods provide different ways to enjoy gameplay. Some have an effect on the score you can achieve during ranked play.",
+                                            TextSize = 20,
+                                        },
+                                        new SpriteText
+                                        {
+                                            Text = @"Others are just for fun",
+                                            TextSize = 20,
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        // main container
+                        new FlowContainer
+                        {
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            RelativeSizeAxes = Axes.Both,
+                            Width = padded_width,

```

Comment:
rather than specifying this each time, why not move the anchor/origin/width to the flowcontainer itself?